docs/configuration.md

```diff
@@ -59,14 +60,15 @@ Property
-**envoy-control.envoy.snapshot.outgoingPermissions.servicesAllowedToUseWildcard**           | Services that are allowed to have the special value in outgoing.dependency field                                                                                                          | false
+Property                                                                                         | Description                                                                                                                                                                               | Default value
+------------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------
+**envoy-control.envoy.snapshot.incomingPermissions.enabled**                                     | Enable incoming permissions                                                                                                                                                               | false
+**envoy-control.envoy.snapshot.incomingPermissions.endpoint-unavailable-status-code**            | Status code when endpoint is not available                                                                                                                                                | 503
+**envoy-control.envoy.snapshot.incomingPermissions.client-identity-header**                      | Header that identifies the client that called the endpoint                                                                                                                                | x-service-name
+**envoy-control.envoy.snapshot.outgoingPermissions.enabled**                                     | Enable outgoing permissions                                                                                                                                                               | false
+**envoy-control.envoy.snapshot.outgoingPermissions.all-services-dependencies-value**             | Special value (wildcard) that signifies that the service depends on all other services                                                                                                    | *
+**envoy-control.envoy.snapshot.outgoingPermissions.services-allowed-to-use-wildcard**            | Services that are allowed to have wildcard in outgoing.dependency field                                                                                                                   | false
+**envoy-control.envoy.snapshot.outgoingPermissions.services-not-included-in-wildcard-by-prefix** | Services not included in dependencies for services with wildcard in outgoing.dependency field. Matched by service name prefix.                                                            | empty list
```

Comment:
Unfortunately no. I planned to something like this to work. But it will slightly complicate the code and I decided it probably wouldn't be needed, so I abandoned the idea. We can also implement it later. Let me know what you think.

Reply:
That's fine, we should just create an issue for it / add it to docs.